0MI0800256/task01.07.03.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+struct Numbers{
+    int a;
+    int b;
+    int c;
+};
+
+int read()
```

Comment:
The name of the function is rather unrelated to what the function does in reality, don't you agree ?